about/trademark-faq.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar: false
+layout: AboutPage
+title: openHAB Trademark Policy FAQ
+meta:
+  - name: og:title
+    content: openHAB Trademark Policy FAQ
+  - name: og:description
+    content: a vendor and technology agnostic open source automation software for your home
+---
+
+# openHAB Trademark Policy - Frequently Asked Questions
```

Comment:
Are these titles necessary since they already appear in the header?

Reply:
Not really, I have removed them. The fact that I didn't notice the duplication shows that the text in the header is not very prominent and a bit detached from the rest of the page, though.